api/src/pcapi/core/fraud/models.py

```diff
@@ -627,3 +627,16 @@ class BlacklistedDomainName(PcObject, Base, Model):
     dateCreated: datetime.datetime = sa.Column(
         sa.DateTime, nullable=False, default=datetime.datetime.utcnow, server_default=sa.func.now()
     )
+
+
+class ProductWhitelist(PcObject, Base, Model):
+    """
+    Add a normally not eligible EAN into a product whitelist
```

Comment:
Ça serait une bonne description de la méthode qui ajoute un EAN à la whitelist, mais pas vraiment de la table...
Qque chose comme "Contains the whitelisted EAN"